maggy/maggy/services/competitor.py

```
@@ -0,0 +1,398 @@
+                    cursor TEXT NOT NULL
+                )
+            """)
+
+    # ── Registry ─────────────────────────────────────────────────────────
+
+    def load_registry(self) -> dict[str, dict]:
+        if not self.competitors_path.exists():
+            return {}
+        try:
+            return json.loads(self.competitors_path.read_text())
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)